src/mono/browser/runtime/invoke-js.ts

```diff
@@ -303,11 +309,16 @@ function bind_fn(closure: BindingClosure) {
             marshal_exception_to_cs(<any>args, ex);
         }
         finally {
+            if (is_async) {
+                Module._free(args as any);
```

Comment:
can args be `NULL` here? I'm guessing that would only be valid for a `fn (void) => void` signature which would go through 0V?

Reply:
No, NULL `args` would be bug in any case here.